articles/native-platforms/ios-objc/02-custom-login.md

```diff
@@ -0,0 +1,199 @@
+  pkgType: 'replace'
+}) %>
+
+### Setting up Auth0
+
+<div class="setup-callback">
+<p> Go to the <a href="${uiAppSettingsURL}">Application Settings</a> part of the dashboard and check that this scheme is set on the *Allowed Callback URLs*:</p>
+
+```
+a0${acount.clientId}://\*.auth0.com/authorize*
+```
+</div>
+
+### 1. Import the Auth0 Toolkit to the project
+
+Your first step is to add [Auth0 Swift Toolkit](https://github.com/auth0/Auth0.swift) into your project, which is a library that contains helper functions that will allow you to perform basic authentication operations (such as login, sign up, or retrieve and update profiles) against the Auth0 API in a very easy manner.
+This toolkit is writen in Swift, but you can incorporate it on your Objective C project just fine. XCode will automatically generate a header file you can include in your source and will enable you to use it's classes and methods as if they were native Objective C code.
+
+#### i. Carthage
+
+If you are using Carthage, add the following line to the `Cartfile`:
+
+```ruby
+github "auth0/Auth0.swift" "1.0.0-beta.3"
+```
+
+Then, run `carthage bootstrap`.
+
+> For more information about Carthage usage, check [their official documentation](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos).
+
+#### ii. Cocoapods
+
+If you are using [Cocoapods](https://cocoapods.org/), add these lines to your `Podfile`:
+
+```ruby
+use_frameworks!
+pod 'Auth0', '1.0.0-beta.3'
+```
+
+Remember to add the `use_frameworks!` line, since it's necessary for Swift pods to be included on XCode even if it's an Objective C project.
+Then, run `pod install`.
+
+> For further reference on Cocoapods, check [their official documentation](http://guides.cocoapods.org/using/getting-started.html).
+
+### 2. Set up your Credentials
+
+Create a property list file named `Auth0.plist` in your project, and add the following entries into it:
+
+| Key       | Value                |
+-----------------------------------
+| ClientId  | ${account.clientId}  |
+| Domain    | ${account.namespace} |
+
+Then you'll need to create a `Auth0InfoHelper` class to handle this properties. With the following methods:
+
+```objc
++ (NSDictionary*) readAuth0Plist
+{
+    NSDictionary *dict = [[NSDictionary alloc] initWithContentsOfFile:[[NSBundle mainBundle] pathForResource:@"Auth0" ofType:@"plist"]];
+
+    return dict;
+}
+```
+
+This will open the `Auth0.plist` file and load it in a dictionary. Then you'll need to create a method to read each value:
+
+```objc
++ (NSString*) Auth0ClientID
+{
+    return [[Auth0InfoHelper readAuth0Plist] objectForKey:@"ClientId"];
+}
+
++ (NSURL*) Auth0Domain
+{
+    return [NSURL a0_URLWithDomain: [[Auth0InfoHelper readAuth0Plist] objectForKey:@"Domain"]];
+}
+```
+
+In the Domain case, we call an extension of `NSURL` included in the `Auth0 Toolkit` that makes sure the URL we are instanciating has a `https` scheme included.
```

Comment:
I'd replace `Auth0 Toolkit` not to use backticks, but rather a link to the library. Like this: [Auth0 Toolkit](https://github.com/auth0/Auth0.swift). Or just remove the backticks, as it's neither code, nor a keyword.